packages/gafl-webapp-service/src/pages/renewals/identify/__tests__/identity.spec.js

```diff
@@ -6,29 +7,45 @@ import {
   TEST_TRANSACTION,
   RENEWAL_PUBLIC,
   RENEWAL_INACTIVE,
-  LICENCE_LENGTH
+  LICENCE_LENGTH,
+  CONTACT_SUMMARY
 } from '../../../../uri.js'
-import { start, stop, initialize, injectWithCookies } from '../../../../__mocks__/test-utils-system.js'
-import { dobHelper, ADULT_TODAY } from '../../../../__mocks__/test-utils-business-rules.js'
+import { start, stop, initialize, injectWithCookies, mockSalesApi } from '../../../../__mocks__/test-utils-system.js'
 
+import { dobHelper, ADULT_TODAY } from '../../../../__mocks__/test-utils-business-rules.js'
 import { salesApi } from '@defra-fish/connectors-lib'
 import { RENEW_AFTER_DAYS, RENEW_BEFORE_DAYS } from '@defra-fish/business-rules-lib'
 import { authenticationResult } from '../__mocks__/data/authentication-result.js'
-import moment from 'moment'
 import * as constants from '../../../../processors/mapping-constants.js'
 import { hasSenior } from '../../../../processors/concession-helper.js'
-import mockConcessions from '../../../../__mocks__/data/concessions'
+import mockDefraCountries from '../../../../__mocks__/data/defra-country.js'
+
+jest.mock('node-fetch')
+
+mockSalesApi()
+salesApi.countries.getAll = jest.fn(async () => new Promise(resolve => resolve(mockDefraCountries)))
+
+beforeAll(() => {
+  process.env.ANALYTICS_PRIMARY_PROPERTY = 'UA-123456789-0'
+  process.env.ANALYTICS_XGOV_PROPERTY = 'UA-987654321-0'
+})
 
 beforeAll(d => start(d))
 beforeAll(d => initialize(d))
 afterAll(d => stop(d))
 
+afterAll(() => {
+  delete process.env.ANALYTICS_PRIMARY_PROPERTY
+  delete process.env.ANALYTICS_XGOV_PROPERTY
+})
+
 const VALID_RENEWAL_PUBLIC = RENEWAL_PUBLIC.uri.replace('{referenceNumber?}', 'AAAAAA')
 const VALID_RENEWAL_PUBLIC_URI = RENEWAL_PUBLIC.uri.replace('{referenceNumber?}', '')
 
 const dobInvalid = moment().add(1, 'years')
 jest.mock('@defra-fish/connectors-lib')
-salesApi.concessions.getAll.mockImplementation(jest.fn(async () => new Promise(resolve => resolve(mockConcessions))))
+mockSalesApi()
+salesApi.countries.getAll = jest.fn(async () => new Promise(resolve => resolve(mockDefraCountries)))
```

Comment:
Why're you mocking this twice? And it'd be slightly more succinct if you did jest.fn(() => Promise.resolve(mockDefraCountries))

Reply:
Eagle-eyed! Thanks.